.travis.yml

```diff
@@ -6,6 +6,7 @@ branches:
   only:
     - master
     - /^build\/.*$/
+    - cs-submit-docker
```

Comment:
This is temporary; I am trying to get travis to build this branch. I will remove this before we merge.

Reply:
Removed